app/assets/stylesheets/styles.scss

```diff
@@ -652,3 +652,9 @@ body {
   background-color: red;
   border-radius: 10px;
 }
+
+form.new_user {
+  input[type=checkbox] {
+    margin-left: 0;
+  }
+}
```

Comment:
This was to fix the alignment of the 'Remember me' checkbox on the login page, but I'm not sure if the same issue is evident elsewhere. Anyway, selecting this via a specific form is probably not the way to go, but I didn't want to mess up anything else, and the alignment was bugging me. :)